traitsui/tests/editors/test_instance_editor.py

```diff
@@ -185,16 +205,12 @@ def test_custom_editor_with_selection_change_option_name(self):
             # test that the current object is None
             self.assertIsNone(obj.inst)
 
-            # test that the displayed text is correct
-            instance = tester.find_by_name(ui, "inst")
-            text = instance.inspect(SelectedText())
-            self.assertEqual(text, obj.inst_list[0].name)
```

Comment:
im not sure why this was removed

Reply:
It was effectively just redundant / already tested elsewhere.  the original issue #1725
was really about the text being correct after changing the name of the selectable object